vevestaX/vevesta.py

```diff
@@ -17,7 +18,16 @@ def __init__(self ):
         self.variables = {}
                 
     def get_filename(self):
-        return ipynbname.name()
+        try:
+            filename = ipynbname.name() + '.ipynb'
+            
+        except:
+            fullPath=sys.argv[0]
+            for i in range(len(fullPath)-1,0,-1):
+                if fullPath[i]=="\\":
+                    filename=fullPath[i+1:]
+                    break
+        return filename
```

Comment:
have 3 cases:
1. jupyter notebook name
2.  sys.argv[0] 
3. filename will be empty so that library doesn't break if case 1 and case 2 fail. In the third case empty filename will be dumped into the file.